src/main/scala/centaur/api/CentaurCromwellClient.scala

```diff
@@ -0,0 +1,52 @@
+package centaur.api
+
+import java.util.concurrent.Executors
+
+import akka.actor.ActorSystem
+import akka.stream.{ActorMaterializer, ActorMaterializerSettings}
+import centaur.CentaurConfig
+import centaur.test.metadata.WorkflowMetadata
+import centaur.test.workflow.Workflow
+import cromwell.api.CromwellClient
+import cromwell.api.model.{CromwellBackends, SubmittedWorkflow, WorkflowStatus}
+
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.concurrent.duration.FiniteDuration
+import scala.util.Try
+
+object CentaurCromwellClient {
```

Comment:
Oh that got my attention.

We should be handing `Future`s back, and not `Try`s I think.